TeamCode/src/main/java/org/firstinspires/ftc/teamcode/util/TowerController.java

```diff
@@ -1,32 +1,54 @@
 package org.firstinspires.ftc.teamcode.util;
 
 import com.qualcomm.robotcore.hardware.DcMotor;
+import com.qualcomm.robotcore.hardware.Servo;
 import com.qualcomm.robotcore.hardware.DigitalChannel;
 import com.qualcomm.robotcore.hardware.HardwareMap;
 import com.qualcomm.robotcore.util.ElapsedTime;
+import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;
 
 import org.firstinspires.ftc.teamcode.TeleOp;
 
-public class TowerController{
+public class TowerController {
 
     public ElapsedTime runtime = new ElapsedTime();
     public DcMotor screw;
+    public DcMotor uBar;
+    public Servo intake;
     public DigitalChannel highSensor;
     public DigitalChannel lowSensor;
+    public DigitalChannel uBarSensor;
     public boolean raiseTower;
+    public boolean intakePos;
+    public double uBarNum = 1;
+
+    public int uBarLevel;
+    static final double     COUNTS_PER_MOTOR    = 384.5;
+    static final double     DRIVE_GEAR_REDUCTION    = 1 ;     // This is < 1.0 if geared UP
+    static final double     COUNTS_PER_INCH         = (COUNTS_PER_MOTOR * DRIVE_GEAR_REDUCTION);
 
     public TowerController (HardwareMap hardwareMap){
 
-        //Setup motor
+        //Setup motors
         screw = hardwareMap.get(DcMotor.class, "screw");
+        uBar = hardwareMap.get(DcMotor.class, "uBar");
+        intake = hardwareMap.get(Servo.class, "intake");
         screw.setDirection(DcMotor.Direction.FORWARD);
+        uBar.setDirection(DcMotor.Direction.FORWARD);
 
         //Setup sensors
         DigitalChannel highSensor = hardwareMap.get(DigitalChannel.class, "highSensor");
         DigitalChannel lowSensor = hardwareMap.get(DigitalChannel.class, "lowSensor");
+        DigitalChannel uBarSensor = hardwareMap.get(DigitalChannel.class, "uBarSensor");
```

Comment:
What's uBarSensor supposed to be? A digital input?

Reply:
It was never used so I removed it.
No Errors